components/automate-cli/pkg/verifyserver/handlers/status/status.go

```diff
@@ -0,0 +1,40 @@
+package status
+
+import (
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/logger"
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/response"
+	"github.com/gofiber/fiber"
+)
+
+type Status struct {
+	Logger logger.ILogger
+}
+
+type IStatus interface {
+	GetStatus(c *fiber.Ctx)
```

Comment:
We will continue adding functions to this interface for every endpoint. What is the need of interface?

Reply:
If testing is the purpose, instead of mocking through interface, using mock server is the better way in this context